deep_quoridor/src/agents/simple.py

```diff
@@ -33,7 +33,7 @@ class SimpleParams(SubargsBase):
     discount_factor: float = 0.99
 
 
-def adjust_wall_position(position: tuple[int, int]) -> tuple[int, int]:
```

Comment:
Thanks for fixing this - I think i have pylance misconfigured because it doesn't complain about types at all.

Reply:
It's nice to have it working, a lot of times the "error" is inocuous like this one but sometimes it points you to a potential bug